Dockerfile.ui_service

```diff
@@ -28,5 +28,6 @@ WORKDIR /root
 RUN /root/services/ui_backend_service/download_ui.sh
 
 RUN pip install --editable .
+RUN pip install git+https://github.com/saikonen/metaflow.git@card-client --upgrade
```

Comment:
remove this once testing is done and card support is completely released with metaflow client.